redis/redis-master-service.yaml

```diff
@@ -1,15 +1,13 @@
+apiVersion: v1
 kind: Service
-apiVersion: v1beta1
-id: redismaster
-# the port that this service should serve on
-port: 10000
-# just like the selector in the replication controller,
-# but this time it identifies the set of pods to load balance
-# traffic to.
-selector:
-  name: redis-master
-# the container on each pod to connect to, can be a name
-# (e.g. 'www') or a number (e.g. 80)
-containerPort: 6379
-labels:
+metadata:
+  labels:
+    name: redis-master
   name: redismaster
+spec:
+  ports:
+    # the port that this service should serve on
+  - port: 6379
+    targetPort: 6379
```

Comment:
Thanks, yeah.  Was thinking to leave it even so, so they could see there was a targetPort setting. Will add a bit more explanation if I do that.